packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/add.controller.js

```diff
@@ -67,6 +66,11 @@ export default class ColdArchiveConfigurationController {
 
   setUsersForContainerCreation() {
     this.users = this.allUserList.filter((user) => user.status === 'ok');
+    this.users.map((user) => {
+      const updatedUser = user;
+      [updatedUser.s3Credentials] = user.s3Credentials;
+      return updatedUser;
+    });
   }
```

Comment:
Question : Why declaring one more variable (updatedUser) ?

Reply:
because `user` is immutable (it's a function parameter).